addons/stock_dropshipping/tests/test_dropship.py

```diff
@@ -121,3 +121,62 @@ def test_00_dropship(self):
             ('location_dest_id', '=', self.env.ref('stock.stock_location_customers').id),
             ('product_id', '=', drop_shop_product.id)])
         self.assertEqual(len(move_line.ids), 1, 'There should be exactly one move line')
+
+    def test_dropship_addresses_report(self):
```

Comment:
> where you can change the dependencies of l10n_it_stock_ddt

All dropship logic is in `stock_dropshipping`, so even if we add the dependencies it won't help

Reply:
@agr-odoo Checking further, I don't think it is needed even to add it as a dependency.  You could actually do the dropshipping test without installing the module by creating the stock.rule/route yourself.  But adding a generic test for in the stock_dropshipping module to fix something for Italy still does not make sense.

Reply:
>  You could actually do the dropshipping test without installing the module by creating the stock.rule/route yourself

I already tried this: we need the `buy` action when defining the rule but it's only added in `purchase_stock` (not installed as dependency)
```python
        self.env['stock.rule'].create({
            'name': '%s → %s' % (supplier_location.name, customer_location.name),
            'action': 'buy',
            'location_id': customer_location.id,
            'location_src_id': supplier_location.id,
            'procure_method': 'make_to_stock',
            'route_id': dropshipping_route.id,
            'picking_type_id': dropship_picking_type.id,
            'company_id': self.env.company.id,
        })
```
Without the full `stock_dropshipping` + `purchase` flow the test would be trivial

> But adding a generic test for in the stock_dropshipping module to fix something for Italy still does not make sense.

I agree, the reason for the move was that a test in `l10n_it_stock_ddt` has a reduced usefulness as it is run only in the nightly builds

All considered we could leave out the test

Reply:
I can agree... but I'll explain my point so I feel more understood 😄.

The e-commerce business in Italy is all about dropshipping DDTs so I fear this kind of relationship between modules will emerge again in the future, so that's why I'd like to have tests on this flow. We already have customers dropshipping Nespresso/Lavazza coffee bags from wholesaler to clients.
(DDTs are an underdeveloped feature at the moment in the Italian localization.)

For now, let's be lean and skip the test, but let's keep that in mind.